qir/qat/TransformationRulesPass/TransformationRulesPassConfiguration.cpp

```diff
@@ -29,10 +29,8 @@ namespace quantum
             assume_no_exceptions_, false, false, "assume-no-except",
             "Assumes that no exception will occur during runtime.");
 
-        config.addExperimentalParameter(
-            reuse_qubits_, false, false, "reuse-qubits", "Use to define whether or not to reuse qubits.");
-        config.addExperimentalParameter(
-            reuse_results_, false, false, "reuse-results", "Use to define whether or not to reuse results.");
+        config.addParameter(reuse_qubits_, false, "reuse-qubits", "Use to define whether or not to reuse qubits.");
```

Comment:
Yes, this means that these are no longer experimental. Arguably the PR text should have been more precise than just stating "updates of settings". That said, note that this does not change the default behaviour of QAT since the default value is set to false. It only moves the option from `experimental` to non-experimental meaning that you can use `--reuse-qubits` without the need for `--experimental`.